stix2elevator/extension_definitions.py

```diff
@@ -0,0 +1,270 @@
+        "name": "Extension to support STIX 1.x Attack Pattern",
+        "description": "This schema adds two properties to a STIX object",
+        "created": "2020-12-22T00:00:00.000000Z",
+        "modified": "2020-12-22T00:00:00.000000Z",
+        "created_by_ref": MITRE_IDENTITY_OBJECT["id"],
+        "object_marking_refs": [
+            MITRE_COPYRIGHT_STATEMENT_DATA_MARKING["id"]
+        ],
+        "schema": "https://https://github.com/mitre/cti/extension-definitions/stix-1-attack-pattern/",
+        "version": "1.0.0",
+        "extension_types": ["property-extension"]
+    },
+    "campaign": {
+        "id": "extension-definition--d83fce45-ef58-4c6c-a3f4-1fbc32e98c6e",
+        "type": "extension-definition",
+        "spec_version": "2.1",
+        "name": "Extension to support STIX 1.x Campaigns",
+        "description": "This schema adds two properties to a STIX object",
+        "created": "2020-12-22T00:00:00.000000Z",
+        "modified": "2020-12-22T00:00:00.000000Z",
+        "created_by_ref": MITRE_IDENTITY_OBJECT["id"],
+        "object_marking_refs": [
+            MITRE_COPYRIGHT_STATEMENT_DATA_MARKING["id"]
+        ],
+        "schema": "https://https://github.com/mitre/cti/extension-definitions/stix-1-campaign/",
+        "version": "1.0.0",
+        "extension_types": ["property-extension"]
+    },
+    "course-of-action": {
+        "id": "extension-definition--a46b18de-0b41-4a95-9d2d-67a360f2d859",
+        "type": "extension-definition",
+        "spec_version": "2.1",
+        "name": "Extension to support STIX 1.x Courses of Action",
+        "description": "This schema adds two properties to a STIX object",
+        "created": "2020-12-22T00:00:00.000000Z",
+        "modified": "2020-12-22T00:00:00.000000Z",
+        "created_by_ref": MITRE_IDENTITY_OBJECT["id"],
+        "object_marking_refs": [
+            MITRE_COPYRIGHT_STATEMENT_DATA_MARKING["id"]
+        ],
+        "schema": "https://https://github.com/mitre/cti/extension-definitions/stix-1-course-of-action/",
+        "version": "1.0.0",
+        "extension_types": ["property-extension"]
+    },
+    "email-message": {
+        "id": "extension-definition--a46b18de-0b41-4a95-9d2d-67a360f2d859",
+        "type": "extension-definition",
+        "spec_version": "2.1",
+        "name": "Extension to support STIX 1.x email-message",
+        "description": "This schema adds two properties to a STIX object",
+        "created": "2020-12-22T00:00:00.000000Z",
+        "modified": "2020-12-22T00:00:00.000000Z",
+        "created_by_ref": MITRE_IDENTITY_OBJECT["id"],
+        "object_marking_refs": [
+            MITRE_COPYRIGHT_STATEMENT_DATA_MARKING["id"]
+        ],
+        "schema": "https://https://github.com/mitre/cti/extension-definitions/stix-1-email-message/",
+        "version": "1.0.0",
+        "extension_types": ["property-extension"]
+    },
+    "icmp-header": {
+        "id": "extension-definition--6f6973cb-70be-40cb-b1ac-15ed78cebe56",
```

Comment:
Calculate new UUID for these: `icmp-header`, `malware` and `network-socket` clash